packages/styletron-core/src/index.js

```diff
@@ -59,7 +61,8 @@ class StyletronCore {
       return cached;
     }
     const virtualCount = this.incrementVirtualCount();
-    const hash = virtualCount.toString(36);
+    const readableClassName = decl.prop + '-' + decl.val;
```

Comment:
I think placing this in a nested ternary would be preferable here so we avoid performing unnecessary string concat